src/scripts/validation.js

```diff
@@ -25,6 +25,19 @@ const hideInputError = (formEl, inputEl, options) => {
 
 // Check input validity
 const checkInputValidity = (formEl, inputEl, options) => {
+  if (inputEl.id === "profile-avatar-input") {
```

Comment:
Please, don't hardcode validation for the `avatar` input inside universal validation code

You should just add `type="url"` for the input in `index.html`